services/stripe-webhook.php

```diff
@@ -425,6 +425,8 @@
 			$logstr .= "Could not find an event with ID #" . $event_id;
 		else
 			$logstr .= "No event ID given.";
+
+		do_action( 'pmpro_webhook_unhandled' );
```

Comment:
Let's add hook doc info via phpdoc for new actions.

```suggestion
/**
 * Allow hooking into after a webhook has been run but was not handled.
 *
 * @since TBD
 *
 * @param string $gateway The gateway the webhook was not handled for.
 */
do_action( 'pmpro_webhook_unhandled', PMPRO_DOING_WEBHOOK );
```